tests/functional/graphql/test_get_slate_lineup.py

```diff
@@ -0,0 +1,84 @@
+from graphql.execution.executors.asyncio import AsyncioExecutor
+from graphene.test import Client
+from fastapi.testclient import TestClient
+
+from app.graphql.graphql import schema
+from app.main import app, load_slate_configs
+from tests.functional.test_dynamodb_base import TestDynamoDBBase
+
+from unittest.mock import patch
+from collections import namedtuple
+
+MockResponse = namedtuple('MockResponse', 'status')
+
+class TestGetSlateLineup(TestDynamoDBBase):
+    client: Client
+
+    async def asyncSetUp(self):
+        await super().asyncSetUp()
+        self.populate_candidate_sets_table()
+        self.client = Client(schema)
+
+    @patch('aiohttp.ClientSession.get', to_return=MockResponse(status=200))
+    def test_get_slate_lineup(self, mock_clientsession_get):
+        with TestClient(
+                app):  # This context manager forces the FastAPI startup event to run, which we use to populate our slate lineup configuration
+            executed = self.client.execute(
+                '''
+                    query GetSlateLineup {
+                      getSlateLineup(slateLineupId: "aaf47c0f-1361-4c8c-a89f-fa45e1dc2978") {
+                        slates {
+                            description
+                        }
+                      }
+                    }
+                ''',
+                context_value={"user_id": "johnjacobjingleheimerschmidt"},
+                executor=AsyncioExecutor())
+
+            # No get defaults here. This chain of 'gets' is deliberately not robust.
+            # We want to test that THIS structure exists in the JSON in its entirety
+            # because our clients depend on this structure.
+            slate_list = executed.get('data').get('getSlateLineup').get('slates')
+
+            # Testing that multiple slates come back
+            assert len(slate_list) > 1
+
+            # Testing that they all have a description
+            assert all('description' in slate for slate in slate_list)
+
+            # FOR YOUR REFERENCE, the response JSON should look something like this:
```

Comment:
Exactly: I imagined the purpose of the test to be to check that a valid request produces a response of the shape that we expect. But if it fails in the future because, for example, we change the wording on one of the slate descriptions, I think that would raise its noise to signal ratio without adding diagnostic benefit.